
diff --git a/thirdparty/qbs/src/packages/chocolatey/chocolatey.qbs b/thirdparty/qbs/src/packages/chocolatey/chocolatey.qbs
index 27cc1e489..ff5409378 100644
--- a/thirdparty/qbs/src/packages/chocolatey/chocolatey.qbs
+++ b/thirdparty/qbs/src/packages/chocolatey/chocolatey.qbs
@@ -14,7 +14,7 @@ Product {
         id: choco
         condition: qbs.targetOS.contains("windows")
         names: ["choco"]
-        platformPaths: { // TODO: Change to platformSearchPaths in qbs 1.15
+        platformSearchPaths: { // TODO: Change to platformSearchPaths in qbs 1.15
             var chocolateyInstall = Environment.getEnv("ChocolateyInstall");
             if (chocolateyInstall)
                 return [FileInfo.joinPaths(chocolateyInstall, "bin")];
